packages/opentelemetry-sdk-trace-base/src/export/BatchSpanProcessorBase.ts

```diff
@@ -45,7 +45,7 @@ export abstract class BatchSpanProcessorBase<T extends BufferConfig>
   private _timer: NodeJS.Timeout | undefined;
   private _shutdownOnce: BindOnceFuture<void>;
   private _droppedSpansCount: number = 0;
-  private _isFlushInProgress: boolean = false;
+  private _flushInProgress: Promise<void> | null = null;
```

Comment:
why _name everywhere?

Reply:
I'm not sure. It's some sort of convention they use in OTEL js, I'm just trying to match the style of the surrounding code.